SDK-1204 Updated docs (#686)
box/box-ios-sdk: docs/usage/metadata.md

To retrieve the collection of available metadata templates in a particular scope, call
[`client.metadata.listEnterpriseTemplates(scope:marker:limit:)`][list-templates]

> with the scope. This method will return an iterator object in the completion that you can use to retrieve metadata templates.

Reviewer: ```suggestion
with the scope. This method will return an iterator object in the completion, which is used to retrieve metadata templates.
```
